src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -100,5 +110,15 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
 
             return builder;
         }
+
+        private static void OnTimedEvent(object state)
```

Comment:
I'm guessing this is for testing only and will be removed?

Reply:
Yeah this is for specializing the host locally and used for manual runs/debugging - will be removed.